litr-demo/src/main/java/com/linkedin/android/litr/demo/data/TransformationPresenter.java

```diff
@@ -671,6 +674,82 @@ public void transcodeToVp9(@NonNull SourceMedia sourceMedia,
                 transformationOptions);
     }
 
+    public void recordAudio(@NonNull AudioRecordMediaSource mediaSource,
+                            @NonNull TargetMedia targetMedia,
+                            @NonNull TransformationState transformationState) {
+        if (targetMedia.targetFile.exists()) {
+            targetMedia.targetFile.delete();
+        }
+
+        transformationState.requestId = UUID.randomUUID().toString();
+        MediaTransformationListener transformationListener = new MediaTransformationListener(context,
+                transformationState.requestId,
+                transformationState,
+                targetMedia);
+
+        try {
+            MediaTarget mediaTarget = new MediaMuxerMediaTarget(targetMedia.targetFile.getPath(),
+                    2,
+                    0,
+                    MediaMuxer.OutputFormat.MUXER_OUTPUT_MPEG_4);
+
+            // Create a single (synthetic) video track to ensure our output is playable by the demo
+            // app. We only use 1 second of video (a solid color) so the duration of the video will
+            // likely depend on the length of the audio recording.
+            VideoTrackFormat videoTrackFormat = new VideoTrackFormat(0, MimeType.VIDEO_AVC);
```

Comment:
To play a video-less video in the Demo app required additional changes. Therefore, I went with a simple approach of adding a short/synthetic video stream.

Reply:
That is totally fine.